### PR TITLE
Don't test futures-nightly feature

### PR DESCRIPTION
**Pull request by EPashkin**
Fix takes too long with no estimate

cc @GuillaumeGomez, @sdroege

### CODE REVIEW

**Comment by EPashkin** [comment]
Now we have many network failures on travis, only one job survived 😭 
Good that it was nightly, that is main reason for this PR.

File: build_travis.sh [hunk 1]
```diff
@@ -6,7 +6,9 @@ set -e
 if [ "$GTK" = latest -o "$GTK" = "3.18" ]; then
 	BUNDLE="gtk-3.18.1-2"
 	if [ "$TRAVIS_RUST_VERSION" = "nightly" ]; then
-		FEATURES=gtk_3_18,futures-nightly
+		FEATURES=gtk_3_18,futures-stable
+# Disabled as nighly not compiles now
```
Comment (sdroege) [inline build_travis.sh]: Typo: nightly

Otherwise please merge :) thanks!

**Comment by sdroege** [comment]
As part of futures and the async/await stuff are in nightly now, I'll take a look at making use of that in the next days. That should hopefully fix things for a while.

**Comment by GuillaumeGomez** [comment]
`crates.io` is down I suppose... But anyway, let's merge this while waiting for a fix from @sdroege.

Thanks @EPashkin!

**Pull request merged**